readme.md

```diff
@@ -0,0 +1,24 @@
+
+# Application Passwords
+
+![](https://cldup.com/50AyeqtUEk.png)
+
+This is a feature plugin that is a spinoff of the main Two-Factor Authentication plugin, found at https://github.com/georgestephanis/two-factor/.  With Application Passwords you are able to authenicate a user without providing his or her password directing, instead you will use a base64 encoded string of their username and a new application password.
```

Comment:
authenicate -> authenticate

"without providing his or her password directing" > "without providing that user's password directly:" (directing -> directly is the operative part but the his or her felt a little clunky to me)